lib/brain.js

```diff
@@ -2454,26 +2527,26 @@ Brain.prototype.isLowBalance = function isLowBalance () {
 }
 
 Brain.prototype.requiredTiers = function requiredTiers (triggers, history, triggerTx) {
-  const triggered = getTriggered(triggers, history, triggerTx)
-  const triggeredFormatted = _.map(o => o.customInfoRequestId ? _.assign(o, { requirement: o.customInfoRequestId }) : o, triggered)
```

Comment:
Why did we drop this formatting? Did the triggers object changed?

Reply:
See comments above. Most of these changes are to standardize trigger behavior